app/flags.go

```diff
@@ -32,6 +33,13 @@ var (
 		Hidden:   true,
 		Required: false,
 	}
+	RetentionFlag = &cli.StringFlag{
+		Name:     RetentionFlagName,
+		Usage:    "The retention of the namespace",
+		Aliases:  []string{"ret"},
```

Comment:
[Nit] tctl uses the alias "rd". Not sure if we want to align on that?
Same with the usage. tctl refers to it as the "workflow execution retention". Might be good to have both tcld and tctl use same description / verbiage if possible?

Reply:
Also, we should mention that the unit is days (even if the name of the flag makes it obvious, nice to also have the usage/help be explicit too) (e.g. "Workflow execution retention in days" or "The retention of the namespace in days")

Reply:
done